agent/ecscni/plugin.go

```diff
@@ -404,6 +404,7 @@ func (client *cniClient) createIPAMNetworkConfig(cfg *Config) (string, *libcni.N
 
 	ipamNetworkConfig := IPAMNetworkConfig{
 		Name:       ECSIPAMPluginName,
+		Type:       ECSIPAMPluginName,
```

Comment:
is type a free string too?

Reply:
what do you mean by "free string"?

Reply:
this field is used to find the plugin binary (as the binary's name)